src/styles/utilities.scss

```
@@ -143,7 +143,7 @@
 @mixin app-layout {
   @include mobile-container;
   @include full-height-layout;
-  background-color: color.$background-color;
+  background-color: color.$white;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)